src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  padding: 0;
```

Comment:
It is not a good idea to use something else besides `box-sizing: border-box;` at `* {}`
```suggestion
```